skills.js

```diff
@@ -1,54 +1,85 @@
 'use strict';
 const constants = require('./constants');
 
+// Effect functions for each skill
+
+//TODO: Real calculations for effects
+var useSlice = function (user, target) {
```

Comment:
Can you use `const` instead of `var` please? It's more consistent with the rest of the code base.